src/Xamarin.Android.Tools.AndroidSdk/JdkInfo.cs

```diff
@@ -45,7 +46,7 @@ public JdkInfo (string homePath)
 			if (homePath == null)
 				throw new ArgumentNullException (nameof (homePath));
 			if (!Directory.Exists (homePath))
-				throw new ArgumentException ("Not a directory", nameof (homePath));
+				throw new ArgumentException ("XamarinAndroidTools_XAT0008", nameof (homePath));
```

Comment:
Shouldn't this use `Resources.ResourceManager.GetString("XamarinAndroidTools_XAT0008")`

Reply:
Yes. Sorry

Reply:
Fixed.

Reply:
Added one more which was missed.